modules/ROOT/pages/sap/sap-connector-config-topics.adoc

```diff
@@ -0,0 +1,171 @@
+= SAP Additional Configuration Information - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+This guide lists additional topics to help you configure the SAP connector.
```

Comment:
Good points. The SAP connector requires that the libraries to be installed to use the connector. It would be best if I moved those sections back to the Studio topic.